src/pages/Send/StepOneForm.js

```diff
@@ -294,23 +310,70 @@ class StepOneForm extends PureComponent<Props, ComponentState> {
     }
   }
 
-  renderSendStepForm = (props: FormRenderProps) => {
+  renderInsufficientFundsError = (hasInsufficientFunds: boolean) => {
     const {
-      handleSubmit,
-      submitting: isSubmitting,
-      valid: isValid,
-      values,
-    } = props
+      i18n,
+      nodeError,
+    }: Props = this.props
+
+    if (nodeError || !hasInsufficientFunds) {
+      return null
+    }
+
+    return (
+      <GlobalFormError
+        text={i18n._(
+          'Send.StepOneForm.global.error.text',
```

Comment:
No need for `global` prefix here. You are not attaching it to any input prefix, so it is already global – related to the whole form scope.